docs/contextual-bandits/actions_contexts.md

```diff
@@ -61,14 +61,13 @@ Contextual Bandits automatically explore new actions as they are encountered and
 
 To create a bandit policy that personalizes, you need to provide context. 
 
-Generally, there are three types of context:
-1. Subject context: For example, the subjects's past behavior, demographic information, are they on a mobile device, etc.
-2. Action context: For example, the size of the discount, the type of product, the price, etc.
```

Comment:
Ah yes this was the cause of this whole documentation update! I assumed action-specific context would be still be useful for the bandit (and it probably would be) but we realized since we build an isolated model per-action and don't model interaction effects, since _every_ user would have the same value for the same attribute they would essentially be constants and useless for personalizing. 

I'll adjust the documentation to make that more clear!